assets/sass/components/first-party-mode/_googlesitekit-first-party-mode-warning-notification.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * First-party mode warning notification styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-fpm-subtle-notification {
+
+		p > a {
+			color: $c-yellow-y-700;
+			text-decoration: underline;
+		}
```

Comment:
This feels like it should be consistent between all notifications of this type so either the designs should be updated to be uniform across other notifications or all links in other notifications should have this underline

Reply:
Also the link should have the external link icon.

Reply:
Updating here from the Slack thread, it would be great to update the base subtle notification styles to take the font color of the main banner style so we can remove this and we can remove this override too:

https://github.com/google/site-kit-wp/blob/9b7f88debff28f274406262a500cb8ecda28172e/assets/sass/components/key-metrics/_googlesitekit-key-metrics-conversion-reporting-subtle-notification.scss#L29-L33

Reply:
Btw, please also remember to use SASS token variables rather than the underlying colour variables, i.e. `$c-utility-on-warning-container` in this case:

https://github.com/google/site-kit-wp/blob/9b7f88debff28f274406262a500cb8ecda28172e/assets/sass/config/_variables-mui3.scss#L170

![image](https://github.com/user-attachments/assets/51c67363-b40d-4a93-9ad4-cbc9dc0af75a)

Reply:
@benbowler @techanvil 

I have changed the styling for warning subtle notification in base notification component styles such that we won't require any separate styles for FPM subtle notification, but I am not modifying anything in ACR notification as of now because although it is a success notification, but it has complete different style, so I think it's better to keep the styles it is for that one.

Also, the external icon can be added later in #9699. I haven't added it here because it was not provided in Figma.